pre-commit

```diff
@@ -28,8 +28,8 @@ for FILE in $FILES; do
     # Ensure the file still exists (i.e. is not being deleted).
     if [ -a $FILE ] ; then
       if [ ${FILE: -3} != ".sh" ] ; then
-        STAT="$(stat -f  "%A" $FILE)"
-        if [ "$STAT" -ne "644" ] ; then
+        STAT="$(stat -c  "%a" $FILE)"
+        if [ "$STAT" -ne "640" ] ; then
```

Comment:
Hmmm... but this works on OSX but also changing the repo to 640 will make EVERY patch fail. Can;'t see how the change to 640 is correct.